common/lib/capa/capa/inputtypes.py

```diff
@@ -629,8 +629,11 @@ def get_attributes(cls):
                                              #   set state
             Attribute('width', "400"),       # iframe width
             Attribute('height', "300"),      # iframe height
-            Attribute('sop', None)           # SOP will be relaxed only if this
-                                             # attribute is set to false.
+            # Title for the iframe, should be supplied by the author of the problem. Not translated
+            # because default display name is also not translated, and we can't import Django here anyway.
+            Attribute('title', "Custom Javascript Display Iframe"),
```

Comment:
Talking to @cptvitamin about this, the default name should be something more like "Course Team Remote Content" or "Problem Remote Content". Screen-reader users will know it is an iFrame, so it doesn't make sense for me to include that.

For our example template, it would be good for the title on the iFrame to match the title element of the contained page. I'll check and see what that is for Cones & Cubes, and keep it in mind when designing the new example.